scripts/reborn_webui_v2_live_qa/run_live_qa.py

```
@@ -1233,19 +1308,124 @@ async def _live_chat_case(
+            ) as response_info:
+                await composer.fill(prompt)  # type: ignore[attr-defined]
+                await composer.press("Enter")  # type: ignore[attr-defined]
+            response = await response_info.value
+        except Exception as exc:
+            if "submission_identity" in observed:
+                raise
+            observed["submission_response_wait_error"] = _exc_text(exc)
+            return False
+        payload = await response.json()
+        if not isinstance(payload, dict):
+            raise AssertionError("chat submission response was not a JSON object")
```

Ruff (0.15.20)

[warning] Prefer `TypeError` exception for invalid type

(TRY004)